Strategy11/formidable-forms
# Introduce Floating Links in the Formidable Admin Panel

This PR introduces the Floating Links feature to the Formidable Admin Panel, which provides users with a set of links for quick navigation to the **Upgrade**, **Support**, and **Documentation** pages.

## QA URL:
https://qa.formidableforms.com/sherv/wp-admin/admin.php?page=formidable

#### NOTE: To test the `PRO version links`, activate the FF PRO plugin.

## Testing Instructions:
1. Navigate to `WP Admin > Formidable`.
2. Check the perfect pixel implementation.
  - **Figma File:** https://www.figma.com/file/YcSWkrjVGroC1zyxhuHH6W/Floating-Links?node-id=1-117&t=Xf5EKO8Mvs1JwuFj-4
3. Verify that all functionality works well.

## Demo - FF Free Version:
![2023-05-09 18 05 47](https://github.com/Strategy11/formidable-forms/assets/69119241/d055e438-1ee6-4dc6-84c4-1137b395fb2b)


## Demo - FF PRO Version:
![2023-05-09 18 02 13](https://github.com/Strategy11/formidable-forms/assets/69119241/81e7d91f-b45e-4be4-a6d0-831f060304c2)
